src/zipAll.js

```diff
@@ -0,0 +1,39 @@
+var _curry1 = require('./internal/_curry1');
+
+/**
+* Creates a new list out of the lists supplied by matching
+* equally positioned items from both lists. The returned list is
+* truncated to the length of the shorter of the two input lists.
+*
+* @func
+* @memberOf R
+* @category List
+* @sig [a] -> [a]
```

Comment:
Actually `[[*]] -> [[*]]`

I don't think there's any way in the type signature to denote that the lists ideally should have the same length, though.